api/app/Models/PoolCandidateSearchRequest.php

```diff
@@ -87,6 +87,49 @@ public static function scopeSearchRequestStatus(Builder $query, ?array $searchRe
         return $query;
     }
 
+    public static function scopeStreams(Builder $query, ?array $streams): Builder
+    {
+        if (empty($streams)) {
+            return $query;
+        }
+
+        // streams is an array of PoolStream enums
+        $query->whereHas('applicantFilter', function ($query) use ($streams) {
+            $query->where(function ($query) use ($streams) {
+                foreach ($streams as $index => $stream) {
+                    $query->orWhereJsonContains('qualified_streams', $stream);
+                }
+            });
+        });
+        return $query;
+    }
```

Comment:
This does not seem to work for me but I'm not sure why 🤔 I selected some streams I _knew_ were part of requests but it always returns empty for me. 😢